lookout/style/typos/generation.py

```diff
@@ -293,7 +295,7 @@ def _vec(self, token: str) -> numpy.ndarray:
         return self.wv[token]
 
     def _freq(self, token: str) -> float:
-        return float(self.frequencies.get(token, self.min_freq))
+        return float(self.frequencies.get(token, 0))
```

Comment:
was it a bug?

Reply:
No, it was a feature. In new conditions using 0 works better for quality.

Reply:
but why do you delete `self.min_freq`? in other conditions, it can be vice-versa. Let's just set to 0.

Reply:
Ok, will do that